cmake/modules/gluecodium/swift/FrameworkInfoPlist.cmake

```diff
@@ -48,6 +72,20 @@ function(apigen_swift_framework_info_plist target)
   get_target_property(SWIFT_RESOURCES_DIR ${target} APIGEN_SWIFT_RESOURCES_DIR)
   get_target_property(SWIFT_FRAMEWORK_NAME ${target} APIGEN_SWIFT_FRAMEWORK_NAME)
   get_target_property(MACOSX_FRAMEWORK_IDENTIFIER ${target} APIGEN_SWIFT_FRAMEWORK_IDENTIFIER)
+  if(IOS)
+    if(CMAKE_OSX_ARCHITECTURES) # Support CMake's default toolchain
+      apigen_check_only_listed_ios_architectures_are_enabled (_is_iphone_os "armv7" "armv7s" "arm64" "arm64e")
```

Comment:
Will this work for simulator? It has "x64" architecture, right?

Reply:
This function detects if all architectures listen in CMAKE_OSX_ARCHITECTURES are architectures for device. If not - below next branch will set APPLE_PLATFORM_NAME to simulator. So yes, this logic works for simulators